File: Python/CheckPrime.py
#!/usr/bin/env python3
import math
def TestPrime(InputInt):
	for DivFactor in range(2, int(math.sqrt(InputInt))+1):
		print(DivFactor)
		if InputInt % DivFactor == 0:
			print("Not Prime")
			return
			
	print("Prime")
	
TestPrime(18907111)

File: Swift/CheckPrime.swift
import Foundation
class Solution {
    func testPrime(_ input: Int) {
        let intRoot = Int(sqrt(Double(input)) + 1)
        for divFactor in 2..<intRoot {
            print(divFactor)
            if input % divFactor == 0 {
                print("Not Prime")
                return
            }
        }
        print("Prime")
    }
}
let solution = Solution()
solution.testPrime(1890711089657)
